Error: Failed to find nvcc when compile version 3.3.2 and 3.3.3

### Details

When I compile with Intel OneAPI suites and CMAKE, version 3.3.0 can be compiled successfully, but versions 3.3.2 and 3.3.3 encounter some problems. 
CMakeConfigureLog.yaml informed me that it **failed to find nvcc**. However, when I attempted with **cmake -B build -DUSE_CUDA=0**, it also failed. Currently, I only need to perform CPU calculations. Could you assist me in solving this compilation issue?
The CMAKE parameters, screen output when compiling, and CMakeConfigureLog.yaml are as follows:

**CMAKE parameters**
> cmake -B build \
> -DCMAKE_CXX_COMPILER=$CMAKE_CXX_COMPILER \
> -DMPI_CXX_COMPILER=$MPI_CXX_COMPILER \
> -DCMAKE_INSTALL_PREFIX=$CMAKE_INSTALL_PREFIX \
> -DELPA_DIR=$ELPA_DIR \
> -DELPA_LIBRARY=$ELPA_LIBRARY \
> -DCEREAL_INCLUDE_DIR=$CEREAL_INCLUDE_DIR \
> -DLibxc_DIR=$Libxc_DIR \
> -DENABLE_LIBXC=ON \
> -DUSE_OPENMP=ON \
> -DENABLE_LIBRI=ON \
> -DGIT_SUBMODULE=OFF \
> -DLIBRI_DIR=$LIBRI_DIR \
> -DLIBCOMM_DIR=$LIBCOMM_DIR \

**screen output when compiling**
> [ 97%] Building CXX object source/module_io/CMakeFiles/io_basic.dir/file_reader.cpp.o
> [ 97%] Building CXX object source/module_io/CMakeFiles/io_basic.dir/csr_reader.cpp.o
> /home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/source/module_io/csr_reader.cpp:112:1: warning: non-void function does not return a value in all control paths [-Wreturn-type]
> }
> ^
> /home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/source/module_io/csr_reader.cpp:136:16: note: in instantiation of member function 'ModuleIO::csrFileReader<double>::getMatrix' requested here
> template class csrFileReader<double>;
>                ^
> 2 warnings generated.
> [ 97%] Built target hamilt_pwdft
> [ 97%] Built target base
> 1 warning generated.
> [ 97%] Built target elecstate
> [ 97%] Built target io_advanced
> 3 warnings generated.
> [ 97%] Built target io_basic
> 1 warning generated.
> [ 97%] Built target ri
> gmake[1]: *** [source/module_hamilt_lcao/hamilt_lcaodft/CMakeFiles/hamilt_lcao.dir/all] Error 2
> gmake[1]: *** Waiting for unfinished jobs....
> [ 97%] Built target esolver
> gmake: *** [all] Error 2
> -- Install configuration: ""
> CMake Error at build/cmake_install.cmake:51 (file):
>   file INSTALL cannot find
>   "/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/abacus": No
>   such file or directory.
>

**CMakeConfigureLog.yaml** 
>     kind: "message-v1"
>     backtrace:
>       - "/home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/share/cmake-3.27/Modules/FindOpenMP.cmake:262 (message)"
>       - "/home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/share/cmake-3.27/Modules/FindOpenMP.cmake:483 (_OPENMP_GET_FLAGS)"
>       - "CMakeLists.txt:187 (find_package)"
>     message: |
>       Parsed CXX OpenMP implicit link information from above output:
>         link line regex: [^( *|.*[/\\])(ld|CMAKE_LINK_STARTFILE-NOTFOUND|([^/\\]+-)?ld|collect2)[^/\\]*( |$)]
>         ignore line: [Change Dir: '/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-EWf51N']
>         ignore line: []
>         ignore line: [Run Build Command(s): /home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/bin/cmake -E env VERBOSE=1 /usr/bin/gmake -f Makefile cmTC_a453c/fast]
>         ignore line: [/usr/bin/gmake  -f CMakeFiles/cmTC_a453c.dir/build.make CMakeFiles/cmTC_a453c.dir/build]
>         ignore line: [gmake[1]: Entering directory `/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-EWf51N']
>         ignore line: [Building CXX object CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o]
>         ignore line: [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin/icpx   -Wno-write-strings -Wno-tautological-constant-compare -fiopenmp -v -std=gnu++11 -MD -MT CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o -MF CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o.d -o CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o -c /home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-EWf51N/OpenMPTryFlag.cpp]
>         ignore line: [Intel(R) oneAPI DPC++/C++ Compiler 2023.1.0 (2023.1.0.20230320)]
>         ignore line: [Target: x86_64-unknown-linux-gnu]
>         ignore line: [Thread model: posix]
>         ignore line: [InstalledDir: /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm]
>         ignore line: [Configuration file: /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../bin/icpx.cfg]
>         ignore line: [Found candidate GCC installation: /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0]
>         ignore line: [Selected GCC installation: /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0]
>         ignore line: [Candidate multilib: .]
>         ignore line: [@m64]
>         ignore line: [Selected multilib: .]
>         ignore line: [@m64]
>         ignore line: [ (in-process)]
>         ignore line: [ "/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/clang++" -cc1 -triple x86_64-unknown-linux-gnu -emit-obj --mrelax-relocations -disable-free -clear-ast-before-backend -disable-llvm-verifier -discard-value-names -main-file-name OpenMPTryFlag.cpp -mrelocation-model static -fveclib=SVML -mframe-pointer=none -menable-no-infs -menable-no-nans -fapprox-func -funsafe-math-optimizations -fno-signed-zeros -mreassociate -freciprocal-math -fdenormal-fp-math=preserve-sign preserve-sign -ffp-contract=fast -fno-rounding-math -ffast-math -ffinite-math-only -mconstructor-aliases -funwind-tables=2 -target-cpu x86-64 -mllvm -x86-enable-unaligned-vector-move=true -tune-cpu generic -mllvm -treat-scalable-fixed-error-as-warning -debugger-tuning=gdb -v -fcoverage-compilation-dir=/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-EWf51N -resource-dir /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib/clang/16 -dependency-file CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o.d -MT CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o -sys-header-deps -I/home/zhuxd/intel/oneapi/tbb/2021.9.0/env/../include -I/home/zhuxd/intel/oneapi/mpi/2021.9.0//include -I/home/zhuxd/intel/oneapi/mkl/2023.1.0/include -I/home/zhuxd/intel/oneapi/ipp/2021.8.0/include -I/home/zhuxd/intel/oneapi/ippcp/2021.7.0/include -I/home/zhuxd/intel/oneapi/ipp/2021.8.0/include -I/home/zhuxd/intel/oneapi/dpl/2022.1.0/linux/include -I/home/zhuxd/intel/oneapi/dpcpp-ct/2023.1.0/include -I/home/zhuxd/intel/oneapi/dnnl/2023.1.0/cpu_dpcpp_gpu_dpcpp/include -I/home/zhuxd/intel/oneapi/dev-utilities/2021.9.0/include -I/home/zhuxd/intel/oneapi/dal/2023.1.0/include -I/home/zhuxd/intel/oneapi/ccl/2021.9.0/include/cpu_gpu_dpcpp -cxx-isystem /home/zhuxd/intel/oneapi/clck/2021.7.3/include -internal-isystem /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/include -internal-isystem /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../include/c++/13.1.0 -internal-isystem /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../include/c++/13.1.0/x86_64-pc-linux-gnu -internal-isystem /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../include/c++/13.1.0/backward -internal-isystem /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib/clang/16/include -internal-isystem /usr/local/include -internal-isystem /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../x86_64-pc-linux-gnu/include -internal-externc-isystem /include -internal-externc-isystem /usr/include -O2 -Wno-write-strings -Wno-tautological-constant-compare -std=gnu++11 -fdeprecated-macro -fdebug-compilation-dir=/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-EWf51N -ferror-limit 19 -fheinous-gnu-extensions -fopenmp-late-outline -fopenmp-threadprivate-legacy -fopenmp -fgnuc-version=4.2.1 -no-opaque-pointers -fcxx-exceptions -fexceptions -vectorize-loops -vectorize-slp -mllvm -paropt=31 -D__GCC_HAVE_DWARF2_CFI_ASM=1 -fintel-compatibility -fintel-libirc-allowed -mllvm -disable-hir-generate-mkl-call -mllvm -loopopt=1 -floopopt-pipeline=light -mllvm -intel-abi-compatible=true -o CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o -x c++ /home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-EWf51N/OpenMPTryFlag.cpp]
>         ignore line: [clang -cc1 version 16.0.0 based upon LLVM 16.0.0git default target x86_64-unknown-linux-gnu]
>         ignore line: [ignoring nonexistent directory "/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../x86_64-pc-linux-gnu/include"]
>         ignore line: [ignoring nonexistent directory "/include"]
>         ignore line: [ignoring duplicate directory "/home/zhuxd/intel/oneapi/ipp/2021.8.0/include"]
>         ignore line: [#include "..." search starts here:]
>         ignore line: [#include <...> search starts here:]
>         ignore line: [ /home/zhuxd/intel/oneapi/tbb/2021.9.0/env/../include]
>         ignore line: [ /home/zhuxd/intel/oneapi/mpi/2021.9.0//include]
>         ignore line: [ /home/zhuxd/intel/oneapi/mkl/2023.1.0/include]
>         ignore line: [ /home/zhuxd/intel/oneapi/ipp/2021.8.0/include]
>         ignore line: [ /home/zhuxd/intel/oneapi/ippcp/2021.7.0/include]
>         ignore line: [ /home/zhuxd/intel/oneapi/dpl/2022.1.0/linux/include]
>         ignore line: [ /home/zhuxd/intel/oneapi/dpcpp-ct/2023.1.0/include]
>         ignore line: [ /home/zhuxd/intel/oneapi/dnnl/2023.1.0/cpu_dpcpp_gpu_dpcpp/include]
>         ignore line: [ /home/zhuxd/intel/oneapi/dev-utilities/2021.9.0/include]
>         ignore line: [ /home/zhuxd/intel/oneapi/dal/2023.1.0/include]
>         ignore line: [ /home/zhuxd/intel/oneapi/ccl/2021.9.0/include/cpu_gpu_dpcpp]
>         ignore line: [ /home/zhuxd/intel/oneapi/clck/2021.7.3/include]
>         ignore line: [ /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/include]
>         ignore line: [ /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../include/c++/13.1.0]
>         ignore line: [ /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../include/c++/13.1.0/x86_64-pc-linux-gnu]
>         ignore line: [ /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../include/c++/13.1.0/backward]
>         ignore line: [ /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib/clang/16/include]
>         ignore line: [ /usr/local/include]
>         ignore line: [ /usr/include]
>         ignore line: [End of search list.]
>         ignore line: [Linking CXX executable cmTC_a453c]
>         ignore line: [/home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/bin/cmake -E cmake_link_script CMakeFiles/cmTC_a453c.dir/link.txt --verbose=1]
>         ignore line: [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin/icpx  -Wno-write-strings -Wno-tautological-constant-compare -fiopenmp -v CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o -o cmTC_a453c  -v ]
>         ignore line: [Intel(R) oneAPI DPC++/C++ Compiler 2023.1.0 (2023.1.0.20230320)]
>         ignore line: [Target: x86_64-unknown-linux-gnu]
>         ignore line: [Thread model: posix]
>         ignore line: [InstalledDir: /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm]
>         ignore line: [Configuration file: /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../bin/icpx.cfg]
>         ignore line: [Found candidate GCC installation: /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0]
>         ignore line: [Selected GCC installation: /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0]
>         ignore line: [Candidate multilib: .]
>         ignore line: [@m64]
>         ignore line: [Selected multilib: .]
>         ignore line: [@m64]
>         link line: [ "/usr/bin/ld" --hash-style=gnu --eh-frame-hdr -m elf_x86_64 -dynamic-linker /lib64/ld-linux-x86-64.so.2 -o cmTC_a453c /lib/../lib64/crt1.o /lib/../lib64/crti.o /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/crtbegin.o -L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/lib/intel64_lin -L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../lib -L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/lib/intel64_lin -L/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0 -L/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../lib64 -L/lib/../lib64 -L/usr/lib/../lib64 -L/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../.. -L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../lib -L/lib -L/usr/lib -L/home/zhuxd/intel/oneapi/tbb/2021.9.0/env/../lib/intel64/gcc4.8 -L/home/zhuxd/intel/oneapi/mpi/2021.9.0//libfabric/lib -L/home/zhuxd/intel/oneapi/mpi/2021.9.0//lib/release -L/home/zhuxd/intel/oneapi/mpi/2021.9.0//lib -L/home/zhuxd/intel/oneapi/mkl/2023.1.0/lib/intel64 -L/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64 -L/home/zhuxd/intel/oneapi/ippcp/2021.7.0/lib/intel64 -L/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64 -L/home/zhuxd/intel/oneapi/dnnl/2023.1.0/cpu_dpcpp_gpu_dpcpp/lib -L/home/zhuxd/intel/oneapi/dal/2023.1.0/lib/intel64 -L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/compiler/lib/intel64_lin -L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib -L/home/zhuxd/intel/oneapi/clck/2021.7.3/lib/intel64 -L/home/zhuxd/intel/oneapi/ccl/2021.9.0/lib/cpu_gpu_dpcpp -L/home/zhuxd/Software/anaconda3/envs/icu/lib/libfabric CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o -Bstatic -lsvml -Bdynamic -Bstatic -lirng -Bdynamic -lstdc++ -Bstatic -limf -Bdynamic -lm -lgcc_s -lgcc -Bstatic -lirc -Bdynamic -ldl -liomp5 -L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib -lgcc_s -lgcc -lpthread -lc -lgcc_s -lgcc -Bstatic -lirc_s -Bdynamic /home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/crtend.o /lib/../lib64/crtn.o]
>           arg [/usr/bin/ld] ==> ignore
>           arg [--hash-style=gnu] ==> ignore
>           arg [--eh-frame-hdr] ==> ignore
>           arg [-m] ==> ignore
>           arg [elf_x86_64] ==> ignore
>           arg [-dynamic-linker] ==> ignore
>           arg [/lib64/ld-linux-x86-64.so.2] ==> ignore
>           arg [-o] ==> ignore
>           arg [cmTC_a453c] ==> ignore
>           arg [-L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/lib/intel64_lin] ==> dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/lib/intel64_lin]
>           arg [-L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../lib] ==> dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../lib]
>           arg [-L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/lib/intel64_lin] ==> dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/lib/intel64_lin]
>           arg [-L/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0] ==> dir [/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0]
>           arg [-L/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../lib64] ==> dir [/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../lib64]
>           arg [-L/lib/../lib64] ==> dir [/lib/../lib64]
>           arg [-L/usr/lib/../lib64] ==> dir [/usr/lib/../lib64]
>           arg [-L/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../..] ==> dir [/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../..]
>           arg [-L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../lib] ==> dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../lib]
>           arg [-L/lib] ==> dir [/lib]
>           arg [-L/usr/lib] ==> dir [/usr/lib]
>           arg [-L/home/zhuxd/intel/oneapi/tbb/2021.9.0/env/../lib/intel64/gcc4.8] ==> dir [/home/zhuxd/intel/oneapi/tbb/2021.9.0/env/../lib/intel64/gcc4.8]
>           arg [-L/home/zhuxd/intel/oneapi/mpi/2021.9.0//libfabric/lib] ==> dir [/home/zhuxd/intel/oneapi/mpi/2021.9.0//libfabric/lib]
>           arg [-L/home/zhuxd/intel/oneapi/mpi/2021.9.0//lib/release] ==> dir [/home/zhuxd/intel/oneapi/mpi/2021.9.0//lib/release]
>           arg [-L/home/zhuxd/intel/oneapi/mpi/2021.9.0//lib] ==> dir [/home/zhuxd/intel/oneapi/mpi/2021.9.0//lib]
>           arg [-L/home/zhuxd/intel/oneapi/mkl/2023.1.0/lib/intel64] ==> dir [/home/zhuxd/intel/oneapi/mkl/2023.1.0/lib/intel64]
>           arg [-L/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64] ==> dir [/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64]
>           arg [-L/home/zhuxd/intel/oneapi/ippcp/2021.7.0/lib/intel64] ==> dir [/home/zhuxd/intel/oneapi/ippcp/2021.7.0/lib/intel64]
>           arg [-L/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64] ==> dir [/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64]
>           arg [-L/home/zhuxd/intel/oneapi/dnnl/2023.1.0/cpu_dpcpp_gpu_dpcpp/lib] ==> dir [/home/zhuxd/intel/oneapi/dnnl/2023.1.0/cpu_dpcpp_gpu_dpcpp/lib]
>           arg [-L/home/zhuxd/intel/oneapi/dal/2023.1.0/lib/intel64] ==> dir [/home/zhuxd/intel/oneapi/dal/2023.1.0/lib/intel64]
>           arg [-L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/compiler/lib/intel64_lin] ==> dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/compiler/lib/intel64_lin]
>           arg [-L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib] ==> dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib]
>           arg [-L/home/zhuxd/intel/oneapi/clck/2021.7.3/lib/intel64] ==> dir [/home/zhuxd/intel/oneapi/clck/2021.7.3/lib/intel64]
>           arg [-L/home/zhuxd/intel/oneapi/ccl/2021.9.0/lib/cpu_gpu_dpcpp] ==> dir [/home/zhuxd/intel/oneapi/ccl/2021.9.0/lib/cpu_gpu_dpcpp]
>           arg [-L/home/zhuxd/Software/anaconda3/envs/icu/lib/libfabric] ==> dir [/home/zhuxd/Software/anaconda3/envs/icu/lib/libfabric]
>           arg [CMakeFiles/cmTC_a453c.dir/OpenMPTryFlag.cpp.o] ==> ignore
>           arg [-Bstatic] ==> search static
>           arg [-lsvml] ==> lib [svml]
>           arg [-Bdynamic] ==> search dynamic
>           arg [-Bstatic] ==> search static
>           arg [-lirng] ==> lib [irng]
>           arg [-Bdynamic] ==> search dynamic
>           arg [-lstdc++] ==> lib [stdc++]
>           arg [-Bstatic] ==> search static
>           arg [-limf] ==> lib [imf]
>           arg [-Bdynamic] ==> search dynamic
>           arg [-lm] ==> lib [m]
>           arg [-lgcc_s] ==> lib [gcc_s]
>           arg [-lgcc] ==> lib [gcc]
>           arg [-Bstatic] ==> search static
>           arg [-lirc] ==> lib [irc]
>           arg [-Bdynamic] ==> search dynamic
>           arg [-ldl] ==> lib [dl]
>           arg [-liomp5] ==> lib [iomp5]
>           arg [-L/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib] ==> dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib]
>           arg [-lgcc_s] ==> lib [gcc_s]
>           arg [-lgcc] ==> lib [gcc]
>           arg [-lpthread] ==> lib [pthread]
>           arg [-lc] ==> lib [c]
>           arg [-lgcc_s] ==> lib [gcc_s]
>           arg [-lgcc] ==> lib [gcc]
>           arg [-Bstatic] ==> search static
>           arg [-lirc_s] ==> lib [irc_s]
>           arg [-Bdynamic] ==> search dynamic
>         collapse library dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/lib/intel64_lin] ==> [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/compiler/lib/intel64_lin]
>         collapse library dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../lib] ==> [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib]
>         collapse library dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../compiler/lib/intel64_lin] ==> [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/compiler/lib/intel64_lin]
>         collapse library dir [/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0] ==> [/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0]
>         collapse library dir [/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../../../lib64] ==> [/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib64]
>         collapse library dir [/lib/../lib64] ==> [/lib64]
>         collapse library dir [/usr/lib/../lib64] ==> [/usr/lib64]
>         collapse library dir [/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0/../../..] ==> [/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib]
>         collapse library dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin-llvm/../lib] ==> [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib]
>         collapse library dir [/lib] ==> [/lib]
>         collapse library dir [/usr/lib] ==> [/usr/lib]
>         collapse library dir [/home/zhuxd/intel/oneapi/tbb/2021.9.0/env/../lib/intel64/gcc4.8] ==> [/home/zhuxd/intel/oneapi/tbb/2021.9.0/lib/intel64/gcc4.8]
>         collapse library dir [/home/zhuxd/intel/oneapi/mpi/2021.9.0//libfabric/lib] ==> [/home/zhuxd/intel/oneapi/mpi/2021.9.0/libfabric/lib]
>         collapse library dir [/home/zhuxd/intel/oneapi/mpi/2021.9.0//lib/release] ==> [/home/zhuxd/intel/oneapi/mpi/2021.9.0/lib/release]
>         collapse library dir [/home/zhuxd/intel/oneapi/mpi/2021.9.0//lib] ==> [/home/zhuxd/intel/oneapi/mpi/2021.9.0/lib]
>         collapse library dir [/home/zhuxd/intel/oneapi/mkl/2023.1.0/lib/intel64] ==> [/home/zhuxd/intel/oneapi/mkl/2023.1.0/lib/intel64]
>         collapse library dir [/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64] ==> [/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64]
>         collapse library dir [/home/zhuxd/intel/oneapi/ippcp/2021.7.0/lib/intel64] ==> [/home/zhuxd/intel/oneapi/ippcp/2021.7.0/lib/intel64]
>         collapse library dir [/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64] ==> [/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64]
>         collapse library dir [/home/zhuxd/intel/oneapi/dnnl/2023.1.0/cpu_dpcpp_gpu_dpcpp/lib] ==> [/home/zhuxd/intel/oneapi/dnnl/2023.1.0/cpu_dpcpp_gpu_dpcpp/lib]
>         collapse library dir [/home/zhuxd/intel/oneapi/dal/2023.1.0/lib/intel64] ==> [/home/zhuxd/intel/oneapi/dal/2023.1.0/lib/intel64]
>         collapse library dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/compiler/lib/intel64_lin] ==> [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/compiler/lib/intel64_lin]
>         collapse library dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib] ==> [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib]
>         collapse library dir [/home/zhuxd/intel/oneapi/clck/2021.7.3/lib/intel64] ==> [/home/zhuxd/intel/oneapi/clck/2021.7.3/lib/intel64]
>         collapse library dir [/home/zhuxd/intel/oneapi/ccl/2021.9.0/lib/cpu_gpu_dpcpp] ==> [/home/zhuxd/intel/oneapi/ccl/2021.9.0/lib/cpu_gpu_dpcpp]
>         collapse library dir [/home/zhuxd/Software/anaconda3/envs/icu/lib/libfabric] ==> [/home/zhuxd/Software/anaconda3/envs/icu/lib/libfabric]
>         collapse library dir [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib] ==> [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib]
>         implicit libs: [svml;irng;stdc++;imf;m;gcc_s;gcc;irc;dl;iomp5;gcc_s;gcc;pthread;c;gcc_s;gcc;irc_s]
>         implicit objs: []
>         implicit dirs: [/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/compiler/lib/intel64_lin;/home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/lib;/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib/gcc/x86_64-pc-linux-gnu/13.1.0;/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib64;/lib64;/usr/lib64;/home/zhuxd/Software/src/GCC/install/gcc-13.1.0/lib;/lib;/usr/lib;/home/zhuxd/intel/oneapi/tbb/2021.9.0/lib/intel64/gcc4.8;/home/zhuxd/intel/oneapi/mpi/2021.9.0/libfabric/lib;/home/zhuxd/intel/oneapi/mpi/2021.9.0/lib/release;/home/zhuxd/intel/oneapi/mpi/2021.9.0/lib;/home/zhuxd/intel/oneapi/mkl/2023.1.0/lib/intel64;/home/zhuxd/intel/oneapi/ipp/2021.8.0/lib/intel64;/home/zhuxd/intel/oneapi/ippcp/2021.7.0/lib/intel64;/home/zhuxd/intel/oneapi/dnnl/2023.1.0/cpu_dpcpp_gpu_dpcpp/lib;/home/zhuxd/intel/oneapi/dal/2023.1.0/lib/intel64;/home/zhuxd/intel/oneapi/clck/2021.7.3/lib/intel64;/home/zhuxd/intel/oneapi/ccl/2021.9.0/lib/cpu_gpu_dpcpp;/home/zhuxd/Software/anaconda3/envs/icu/lib/libfabric]
>         implicit fwks: []
>       
>       
>   -
>     kind: "try_compile-v1"
>     backtrace:
>       - "/home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/share/cmake-3.27/Modules/FindOpenMP.cmake:419 (try_compile)"
>       - "/home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/share/cmake-3.27/Modules/FindOpenMP.cmake:557 (_OPENMP_GET_SPEC_DATE)"
>       - "CMakeLists.txt:187 (find_package)"
>     description: "Detecting CXX OpenMP version"
>     directories:
>       source: "/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-BFVZUy"
>       binary: "/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-BFVZUy"
>     cmakeVariables:
>       CMAKE_CXX_FLAGS: " -Wno-write-strings -Wno-tautological-constant-compare"
>       CMAKE_CXX_FLAGS_DEBUG: "-g"
>       CMAKE_EXE_LINKER_FLAGS: ""
>       CMAKE_MODULE_PATH: "/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/cmake"
>     buildResult:
>       variable: "OpenMP_SPECTEST_CXX_"
>       cached: true
>       stdout: |
>         Change Dir: '/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-BFVZUy'
>         
>         Run Build Command(s): /home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/bin/cmake -E env VERBOSE=1 /usr/bin/gmake -f Makefile cmTC_1f383/fast
>         /usr/bin/gmake  -f CMakeFiles/cmTC_1f383.dir/build.make CMakeFiles/cmTC_1f383.dir/build
>         gmake[1]: Entering directory `/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-BFVZUy'
>         Building CXX object CMakeFiles/cmTC_1f383.dir/OpenMPCheckVersion.cpp.o
>         /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin/icpx   -Wno-write-strings -Wno-tautological-constant-compare -fiopenmp -std=gnu++11 -MD -MT CMakeFiles/cmTC_1f383.dir/OpenMPCheckVersion.cpp.o -MF CMakeFiles/cmTC_1f383.dir/OpenMPCheckVersion.cpp.o.d -o CMakeFiles/cmTC_1f383.dir/OpenMPCheckVersion.cpp.o -c /home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-BFVZUy/OpenMPCheckVersion.cpp
>         Linking CXX executable cmTC_1f383
>         /home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/bin/cmake -E cmake_link_script CMakeFiles/cmTC_1f383.dir/link.txt --verbose=1
>         /home/zhuxd/intel/oneapi/compiler/2023.1.0/linux/bin/icpx  -Wno-write-strings -Wno-tautological-constant-compare -fiopenmp CMakeFiles/cmTC_1f383.dir/OpenMPCheckVersion.cpp.o -o cmTC_1f383 
>         gmake[1]: Leaving directory `/home/zhuxd/Software/abacus/repo/abacus-develop-3.3.3/build/CMakeFiles/CMakeScratch/TryCompile-BFVZUy'
>         
>       exitCode: 0
>   -
>     kind: "message-v1"
>     backtrace:
>       - "/home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/share/cmake-3.27/Modules/CheckLanguage.cmake:100 (message)"
>       - "CMakeLists.txt:195 (check_language)"
>     checks:
>       - "Looking for a CUDA compiler"
>     message: |
>       Looking for a CUDA compiler failed with the following output:
>       CMake Error at /home/zhuxd/Software/src/CMAKE/cmake-3.27.0-rc5-linux-x86_64/share/cmake-3.27/Modules/CMakeDetermineCUDACompiler.cmake:180 (message):
>         Failed to find nvcc.
>       
>         Compiler requires the CUDA toolkit.  Please set the CUDAToolkit_ROOT
>         variable.
>       Call Stack (most recent call first):
>         CMakeLists.txt:2 (project)
>       
>       
>       -- Configuring incomplete, errors occurred!
>       
> ...
>

--- a/docs/advanced/install.md
+++ b/docs/advanced/install.md
@@ -271,7 +271,7 @@ directly.
 > `deepmd_c`/`deepmd_cc` and `tensorflow_cc` libraries would be called according to `DeePMD_DIR` and `TensorFlow_DIR`, which is showed in detail in [this page](https://github.com/deepmodeling/deepmd-kit/blob/master/doc/inference/cxx.md).
 
 ### Add LibRI and LibComm Support
-To use new EXX, you need two libraries: LibRI and LibComm and need to define `LIBRI_DIR` and `LIBCOMM_DIR` in the file `Makefile.vars` or use 
+To use new EXX, you need two libraries: [LibRI](https://github.com/abacusmodeling/LibRI) and [LibComm](https://github.com/abacusmodeling/LibComm) and need to define `LIBRI_DIR` and `LIBCOMM_DIR` in the file `Makefile.vars` or use 
 ```makefile
 make LIBRI_DIR=/public/software/LibRI LIBCOMM_DIR=/public/software/LibComm
 ```

